src/conventions/datapack_uninstallation.md

```diff
@@ -11,3 +11,105 @@ There are no rules enforcing how you should implement the uninstallation method,
 ### Uninstallation Function
 
 You can create an "uninstallation function", which when ran, will remove most of the content added from your datapack. You can tell the user about this function inside your project page or anywhere you like.
+
+--------------------
+
+### **Note from Convention Mod (Optional)**
+> I recommend if you have `scoreboard`, `storage`, `team`, you should remove it if possible.  
+> - If you can't remove something, such as `core scoreboard`, you can write the reason in the [request review](https://github.com/mc-datapacks/review-tracker/issues) or mcfunction comment.
+
+###  **What should you know and be aware of about how to disable datapack**
+**1. Use /datapack disable \<name>** - `tick.json`
+> Pros
+> - Using just 2 commands will clear the schedule command as well. 
+> - Auto reload, Help ensure that everything is completely cleared. Due to the use of the `/reload` command, some multiplayer servers have a server launcher. Changed this reload command so vanilla's commands it won't work.

```

Comment:
This is what I want to convey is If using only these two commands
```mcfunction
datapack disable "file/convention"
datapack disable "file/convention.zip"
```
So there's no need to use any `/schedule clear` commands.

Also, using this command will cause reload without using the `/reload` command, as the `/reload` command will have issues with some brand server launchers. If I'm not mistaken, it's probably the "paper". I can't use the "/reload" command to reload the datapack.

This is what I understand. If I misunderstood something, please point out.

Reply:
```suggestion
> - Only use two `datapack disable` commands. This will also disable the schedule command.
> - Automatic reload without `/reload`.  **Note:** `/reload` command doesn't work with some 
server launchers.
```